--- cmath.bak	2005-12-05 06:06:32.000000000 +0200
+++ cmath	2005-12-05 06:07:11.000000000 +0200
@@ -343,21 +343,21 @@
   inline float
   pow(float __x, float __y)
-  { return __builtin_powf(__x, __y); }
+  { return __pow_helper(__x, __y); }
 
   inline long double
   pow(long double __x, long double __y)
-  { return __builtin_powl(__x, __y); }
+  { return __pow_helper(__x, __y); }
 
   inline double
   pow(double __x, int __i)
-  { return __builtin_powi(__x, __i); }
+  { return __pow_helper(__x, __i); }
 
   inline float
   pow(float __x, int __n)
-  { return __builtin_powif(__x, __n); }
+  { return __pow_helper(__x, __n); }
 
   inline long double
   pow(long double __x, int __n)
-  { return __builtin_powil(__x, __n); }
+  { return __pow_helper(__x, __n); }
 
   using ::sin;
